.eslintrc.js

```diff
@@ -32,6 +33,7 @@ module.exports = {
     ],
     'import/no-namespace': 'off',
     'import/no-named-export': 'off',
+    'no-redeclare': 'off',
```

Comment:
that's not the right fix. see https://github.com/rollup/eslint-config-rollup/pull/3/files#r538998040

Reply:
this actually does seem like the right fix to me per the conversation on the thread linked in the above comment